modules/dxp/apps/portal-workflow/portal-workflow-kaleo-metrics-integration/src/main/java/com/liferay/portal/workflow/kaleo/metrics/integration/internal/model/listener/KaleoTaskInstanceTokenModelListener.java

```diff
@@ -64,21 +65,14 @@ public void onAfterCreate(KaleoTaskInstanceToken kaleoTaskInstanceToken)
 							kaleoTaskInstanceToken.
 								getKaleoTaskInstanceTokenId());
 
-				Long[] assigneeIds = Optional.ofNullable(
-					kaleoTaskAssignmentInstances
-				).filter(
-					ListUtil::isNotEmpty
-				).map(
-					List::stream
-				).map(
-					stream -> stream.map(
-						KaleoTaskAssignmentInstance::getAssigneeClassPK
-					).toArray(
-						Long[]::new
-					)
-				).orElseGet(
-					() -> null
-				);
+				Long[] assigneeIds = _getAssigneeInfo(
+					kaleoTaskAssignmentInstances,
+					KaleoTaskAssignmentInstance::getAssigneeClassPK);
+
+				Long[] assigneeGroupIds = _getAssigneeInfo(
+					kaleoTaskAssignmentInstances,
+					KaleoTaskAssignmentInstance::getGroupId);
+
```

Comment:
@seltong, please could you do this changes above in [this commit](https://github.com/liferay-workflow/liferay-portal/pull/452/commits/5c3150ed95043063e63304b8365e99828a912692)?, thanks!